magellan-library/src/main/java/com/wealthfront/magellan/view/DialogComponent.kt

```diff
@@ -39,10 +42,14 @@ public class DialogComponent @Inject constructor() : LifecycleAware {
 
   override fun pause(context: Context) {
     destroyDialog()
+  }
+
+  override fun hide(context: Context) {
     this.context = null
   }
 
   private fun createDialog() {
+    shouldRestoreDialog = false
```

Comment:
will this prevent dialogs from surviving multiple pause/resume runs? Hmm I do see `shouldRestoreDialog` gets set back to true ondestroy

Reply:
[device-2023-02-27-151404.webm](https://user-images.githubusercontent.com/5955888/221673517-09d8a2df-ef1e-43d1-af40-276ffb3245a3.webm)

Reply:
The dialog in this example survives several pause/resume cycles, thanks to `shouldRestoreDialog`